elastic/kibana
# [RAC] integrating rbac search strategy with alert flyout

## Summary

We are integrating alert search strategy with RBAC on top of alert flyout for security solution.

Steps to verify:
1. Please check if the data in these flyouts loaded properly:
<img width="1665" alt="Screenshot 2021-08-12 at 09 35 39" src="https://user-images.githubusercontent.com/6295984/129165740-1d79f2a5-260d-431b-9035-2900881b76cf.png">
<img width="1664" alt="Screenshot 2021-08-12 at 09 34 55" src="https://user-images.githubusercontent.com/6295984/129165744-7597033d-e26c-4ea5-864f-187193e1e266.png">
<img width="1663" alt="Screenshot 2021-08-12 at 09 34 31" src="https://user-images.githubusercontent.com/6295984/129165745-85909769-f5a5-4105-838b-da0040302071.png">
<img width="1661" alt="Screenshot 2021-08-12 at 09 34 11" src="https://user-images.githubusercontent.com/6295984/129165746-fd9862ff-f3bf-4902-adf4-2c5bac49b5fd.png">

2. Please enable tGrid and check them again.
